product_simple_seasonality/models/seasonality.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2024 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Seasonality(models.Model):
+    _name = "seasonality"
+    _description = "Seasonality"
+
+    name = fields.Char(required=True)
+    company_id = fields.Many2one(
+        "res.company", required=True, default=lambda self: self.env.company
```

Comment:
```suggestion
        "res.company", default=lambda self: self.env.company
```

Reply:
@kevinkhao why did you mark as resolved ?

Reply:
dropped committed suggestion while rebasing. corrected now